fiftyone/utils/cvat.py

```diff
@@ -2278,6 +2312,20 @@ def to_eta_attribute(self):
 
         return etad.CategoricalAttribute(self.name, self.value)
 
+    def to_attribute(self):
+        """Returns a :class:`fiftyone.core.labels.Attribute` representation of
+        the attribute.
+        Returns:
+            a :class:`fiftyone.core.labels.Attribute`
+        """
+        if isinstance(self.value, bool):
```

Comment:
We're trying to get away from using the `fiftyone.core.labels.Attribute` classes. I wanted to delete them in recent work I did, but stopped short since `label.attributes` may still be used by some datasets out there.

Can you avoid doing this type conversion?

Reply:
I do still need this to be able to use the quickstart dataset in examples since `iscrowd` is stored as a FiftyOne attribute. Though we should probably update that dataset so that `iscrowd` and `area` are stored directly on labels instead of in attributes